deployment/v3/mosip/pms-migration-utility/README.md

```diff
@@ -1,6 +1,15 @@
 # Partner Management Service 115-120 Migration Utility
 
 ## Install
-```
-$ ./install.sh
-```
+* Run `./install.sh` to deploy pms-migration-utility.
+  ```
+  $ ./install.sh
+  ```
+* The script will prompt to run pms-migration-utility as cronjob,
```

Comment:
The script will prompt to run the `pms-migration-utility` as Cronjob.  If the input provided is `Y`, then the script will deploy `pms-migration-utility` as Cronjob otherwise the script will deploy `pms-migration-utility` as a job.